easybuild/easyconfigs/y/Yade/Yade-2016.06a-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'CMakeMake'
+
+name = 'Yade'
+version = '2016.06a'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://yade-dem.org/doc/'
+description = """ Yade is an extensible open-source framework for discrete numerical models, 
+ focused on Discrete Element Method. The computation parts are written in c++ using flexible object model, 
+ allowing independent implementation of new alogrithms and interfaces. 
+ Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['Yade-%(version)s_include-signal.patch']
+
+dependencies = [
+    ('Boost', '1.61.0', versionsuffix),
+    ('CGAL', '4.8.1', versionsuffix),
+    ('GDB', '7.11.1', versionsuffix),
+    ('GL2PS', '1.3.9'),
+    ('GTS', '0.7.6'),
+    ('IPython', '5.1.0', versionsuffix),
+    ('libQGLViewer', '2.6.3'),
+    ('Loki', '0.1.7'),
+    ('matplotlib', '1.5.3', versionsuffix),
+    ('minieigen', '0.5.3', versionsuffix),
+    ('PyGTS', '0.3.1', versionsuffix),
+    ('Python', '2.7.12'),
+    ('Qt5', '5.7.0'),
+    ('SQLite', '3.13.0'),
+    ('VTK', '7.0.0', versionsuffix),
```

Comment:
@jrbosch Are you sure Yade is compatible with VTK7? The `intel/2016b` easyconfig uses VTK 6.3.0, I seem to recall there was a good reason to stick to VTK6...